lib/dock.rb

```diff
@@ -0,0 +1,47 @@
+require './lib/boat'
+require './lib/renter'
+require 'pry'
+
+class Dock
```

Comment:
Overall, I like your logic, and I can see you working towards solving the problem. You didn't quite get there, but you were really close. Good job!